code/game/mecha/mecha_control_console.dm

```diff
@@ -91,6 +91,12 @@
 			M.trackers -= src
 	return ..()
 
+/obj/item/mecha_parts/mecha_equipment/try_attach_part(mob/user, obj/mecha/M)

```

Comment:
Should this be `mecha_tracking` rather than `mecha_equipment`?

Reply:
yes